tests/e2e/landing.test.ts

```diff
@@ -2,5 +2,7 @@ import { expect, test } from '@playwright/test'
 
 test('Landing page working, we can start 🚀', async ({ page }) => {
   await page.goto('/')
+  await page.waitForResponse((response) => response.status() === 200)
+
   expect(await page.locator('h1').textContent()).toBe('Home')
 })
```

Comment:
I kinda like to have e2e test of pages directly in the `routes` folder. There is a specific reason for this change ?